lib/erb_lint/cache.rb

```diff
@@ -9,6 +9,8 @@ def initialize(config, cache_dir = nil)
       @cache_dir = cache_dir || CACHE_DIRECTORY
       @hits = []
       @new_results = []
+      @gemfile_lock = File.read("Gemfile.lock") if File.exist?("Gemfile.lock")
```

Comment:
Nice, I think this should effectively cache the `File.read` for all calls of `checksum`.